engine/src/main/java/io/seldon/engine/App.java

```diff
@@ -50,16 +52,11 @@ public GracefulShutdown gracefulShutdown() {
     }
     
     @Bean
-    public EmbeddedServletContainerCustomizer tomcatCustomizer() {
-        return new EmbeddedServletContainerCustomizer() {
-
+    public WebServerFactoryCustomizer tomcatCustomizer() {
+        return new WebServerFactoryCustomizer<TomcatServletWebServerFactory>() {
             @Override
-            public void customize(ConfigurableEmbeddedServletContainer container) {
-                if (container instanceof TomcatEmbeddedServletContainerFactory) {
```

Comment:
I wonder was there some reason for the Tomcat check here?

Reply:
Before, the Customizer class was completely type-agnostic, so we received a `ConfigurableEmbeddedServletContainer` instance which we had to downcast to a `TomcatEmbeddedServletContainerFactory`. In this case, it made a lot of sense to check first if we could actually do that.

However, `spring-boot > 2.0.x` changed this a bit so that the Customizer now acts as a template. In our case, we implement a `WebServerFactoryCustomizer<TomcatServletWebServerFactory>`, which is only going to customise `TomcatServletWebServerFactory` instances. Therefore, we don't need to downcast anymore (since the class will only be applied to customize Tomcat instances).